src/module/stereo_splitter.rs

```diff
@@ -35,11 +35,14 @@ impl Module for StereoSplitter {
         None
     }
 
-    fn inputs(&self) -> &[LineType] {
-        &[LineType::Stereo]
+    fn inputs(&self) -> Vec<Terminal> {
+        vec![LineType::Stereo.unlabeled()]
     }
 
-    fn outputs(&self) -> &[LineType] {
-        &[LineType::Mono, LineType::Mono]
+    fn outputs(&self) -> Vec<Terminal> {
+        vec![
+            LineType::Mono.labeled("L"),
+            LineType::Mono.labeled("R")
```

Comment:
These are the only ones I've labeled for the time being (and I haven't even checked that it is L/R and not R/L). I thought if we liked this direction, we could decide on the labeling convention, and I'll update each as necessary.

Reply:
Yep, L/R is correct! 👍